src/app/view/perspectives/post-it/document/post-it-document.component.html

```diff
@@ -37,7 +37,7 @@
               (keydown.enter)="selectionHelper.setEditMode(true); selectionHelper.focus()"
               (click)="clickOnAttributePair(0, row)">
 
-          <input class="border-0"
+          <input class="border-0 {{ isDefaultAttribute(attributePair.attribute) ? 'default-attribute' : null }}"
```

Comment:
Isn't better to use it like `[class.default-attribute]="isDefaultAttribute(attributePair.attribute)"` ?

Reply:
I don't know. Are there any advantages?

If not, we can just choose one and stick to that. As far as I know, we use the `{{ }}` syntax more.

Reply:
I think it does not really matter although I find the syntax suggested by @kubedo8 much cleaner. We use both styles across the application.